hologram_dashboard/proto++/protovalue.h

```diff
@@ -0,0 +1,111 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#pragma once
```

Comment:
Is this used?

Reply:
This is to prevent including the same headers more than once in case other library uses this code

Reply:
In Google, we will use include guard for this:
https://google.github.io/styleguide/cppguide.html#The__define_Guard